### PR TITLE
add test for #18109: findlast with unicode

### PR DESCRIPTION
**Pull request by KristofferC**
Closes https://github.com/JuliaLang/julia/issues/18109

### CODE REVIEW

File: test/strings/search.jl [hunk 1]
```diff
@@ -328,3 +328,7 @@ end
 @test @inferred findall(isequal('a'), "éa") == [3]
 @test @inferred findall(isequal('€'), "€€") == [1, 4]
 @test @inferred isempty(findall(isequal('é'), ""))
+
+# issue #18109
+s_18109 = "fooα🐨βcd3"
```
Comment (JeffBezanson) [inline test/strings/search.jl]: This variable isn't necessary.

Comment (KristofferC) [reply test/strings/search.jl]: Oh yeah, I originally had two tests here but realized one was redundant.

**Comment by KristofferC** [comment]
@stevengj Is this test enough to close #18109 in your opinion?

**Comment by stevengj** [comment]
Yes, the underlying bug seems to have been fixed a while ago (`findprev` now uses `prevind`).  For completeness, it would be good to add
```jl
@test findfirst(isequal('d'), "fooα🐨βcd3") == 13
```
as well.

**Review by stevengj — APPROVED** [review]
